gui/test/integration/cypress/support/page_files/Frame.ts

```diff
@@ -0,0 +1,16 @@
+export abstract class Frame {
```

Comment:
I usually have a Page superclass in preference to mirroring the components. Then Page has frame, which has navbar, etc.